code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -1,39 +1,39 @@
 
-/obj/item/bodypart/proc/can_dismember(obj/item/I)
+/obj/item/bodypart/proc/can_dismember(obj/item/item)
 	if(dismemberable)
 		return TRUE
 
 //Dismember a limb
 /obj/item/bodypart/proc/dismember(dam_type = BRUTE, silent=TRUE)
 	if(!owner || !dismemberable)
 		return FALSE
-	var/mob/living/carbon/C = owner
-	if(C.status_flags & GODMODE)
+	var/mob/living/carbon/limb_owner = owner
+	if(limb_owner.status_flags & GODMODE)
 		return FALSE
-	if(HAS_TRAIT(C, TRAIT_NODISMEMBER))
+	if(HAS_TRAIT(limb_owner, TRAIT_NODISMEMBER))
 		return FALSE
 
-	var/obj/item/bodypart/affecting = C.get_bodypart(BODY_ZONE_CHEST)
+	var/obj/item/bodypart/affecting = limb_owner.get_bodypart(BODY_ZONE_CHEST)
 	affecting.receive_damage(clamp(brute_dam/2 * affecting.body_damage_coeff, 15, 50), clamp(burn_dam/2 * affecting.body_damage_coeff, 0, 50), wound_bonus=CANT_WOUND) //Damage the chest based on limb's existing damage
 	if(!silent)
-		C.visible_message("<span class='danger'><B>[C]'s [name] is violently dismembered!</B></span>")
-	INVOKE_ASYNC(C, /mob.proc/emote, "scream")
-	playsound(get_turf(C), 'sound/effects/dismember.ogg', 80, TRUE)
-	SEND_SIGNAL(C, COMSIG_ADD_MOOD_EVENT, "dismembered", /datum/mood_event/dismembered)
+		limb_owner.visible_message("<span class='danger'><B>[limb_owner]'s [name] is violently dismembered!</B></span>")
+	INVOKE_ASYNC(limb_owner, /mob.proc/emote, "scream")
```

Comment:
do you know why this is async'd?

Reply:
No clue, I'm just scrubbin' out single letter vars

Reply:
emotes can sleep I'd bet

Reply:
I'ma mark this as resolved cause tbh I barely know what async even means but it was there when I cleaned it up and there's a reasonable hypothesis as to why it's there, so it should probably stay unless someone specifically finds that it's not needed